src/huggingface_hub/README.md

```diff
@@ -371,4 +371,38 @@ API.
 
 ```python
 inference = InferenceApi("bert-base-uncased", task="feature-extraction", token=API_TOKEN)
-```
\ No newline at end of file
+```
+
+## Controlling the logging of `huggingface_hub`
```

Comment:
Should this information be in README? How are we going to create the documentation for this library? We probably need quite a few user guides and I think this should go in one of those, which probably should be located in the `/doc` folder.

Reply:
I'm putting this here for now, but this should definitely be in a separate guide as outlined in https://github.com/huggingface/huggingface_hub/issues/754